packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -96,12 +95,11 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable {
         uint256 liveness, // TODO: think about changing this to challenge window?
         bytes32 identifier
     ) public nonReentrant() returns (bytes32) {
-        // TODO: if you want to assert for yourself set this to your own address NOT address(0).
-        asserter = asserter == address(0) ? msg.sender : asserter;
         // TODO: think about placing either msg.sender or block.timestamp into the claim ID to block an advasery
         // creating a claim that collides with a known assertion that will be created into the future.
         bytes32 assertionId = _getId(claim, bond, liveness, currency, callbackRecipient, escalationManager, identifier);
 
+        require(asserter != address(0), "Asserter cant be 0");
```

Comment:
Just to keep consistent interface we should also use the same logic in `disputeAssertionFor` by not allowing caller to provide 0 addrress.

Reply:
that is a great catch! I will update.

what do you think about re-naming `disputeAssertionFor` to just `disputeAssertion` now that we call `assertTruthFor` `assertTruth`?

Reply:
yes, `disputeAssertion` sounds good!